### ISSUE TITLE
Mixed JS code from demo code

### ISSUE THREAD

**Issue by akshaybabloo** [creativetimofficial/ct-material-kit-pro#61]
At line `97 - 102` there is a code 

```js
if (window_width >= 768) {
        big_image = $('.page-header[data-parallax="true"]');
        if (big_image.length != 0) {
            $(window).on('scroll', materialKitDemo.checkScrollForParallax);
        }

    }
```

I think `materialKitDemo` should be `materialKit`

**Comment by dragosct10** [comment]
Hi, @akshaybabloo! Thank you for using our product. We put that function with other functions in assets-for-demo folder because we used these functions only for demo purpose, but of course if are users like you that want to include some of these functions in the project you can do it without any problem. 

Regards,
Dragos

**Comment by akshaybabloo** [comment]
Ya but the problem is that if people like me don't know that this has happened in the main JS file and if I use this in, say, Angular I would get a fatal error!! But that's alright; I fixed manually.

**Comment by dragosct10** [comment]
Hi, @akshaybabloo! Yes, that makes sense. We will change in the next update. 

Regards,
Dragos

**Comment by groovemen** [comment]
Hello @akshaybabloo,

Thank you for your interest in working with our products, this issue is solved in the latest version (v2.1.0).

All the best,
Stefan
